tests/test_followings.py

```diff
@@ -111,14 +145,133 @@ def test_delete_followings(self):
     data = json.loads(response.data)
     self.assertEquals(len(data['data']['followings']), 1)
 
+    response = self.app.get('api/v1/followers/show/{}/'.format(test_user_id1))
+    data = json.loads(response.data)
+    self.assertEquals(len(data['data']['followers']), 0)
+
+    response = self.app.get('api/v1/followers/show/{}/'.format(test_user_id2))
+    data = json.loads(response.data)
+    self.assertEquals(len(data['data']['followers']), 1)
+
+    response = self.app.get('api/v1/followings/show/{}/'.format(test_user_id2))
+    data = json.loads(response.data)
+    self.assertEquals(len(data['data']['followings']), 0)
+
```

Comment:
Aren't these kind of redundant? This behavior should have already been checked in the other two tests and not delete

Reply:
idk dude tbh I was just messing around with it trying to find the bug, and I thought it wouldn't hurt to just put it into master since I've already done the work lol